### DISCUSSION TITLE
getantidote/antidote ?

### DISCUSSION THREAD

**Discussion by segevfiner** [mattmc3/antidote#190] [General]
The current install instruction seem to point to cloning https://github.com/mattmc3/antidote which always gets the latest commit, but I encountered https://github.com/getantidote/antidote which seems to be a fork that only has the stable versions? Are the instructions to clone mattmc3/antidote and always get the latest version the right ones? Or did you intend users to use the stable fork? Or is that fork deprecated?

**Comment by mattmc3** [comment]
They should nearly always be in sync after a release bakes in. It's definitely a maintained fork, and is only for people that really, really don't ever want to risk a brand new release. `mattmc3/antidote` is the primary home for this repo, and where I want most users to be, so the other remains mostly undocumented.

Perhaps one day it might make sense to move antidote from `mattmc3/antidote` to the `getantidote/antidote` - one reason might be if we have bunches of maintainers, but for right now it's just me (for both places). `mattmc3/antidote` is where development and new releases happen and will continue to happen for the foreseeable future. About a week or two after a release, if no major issues crop up, I sync the fork.
